diff --git a/src/backend/graph_compiler/core/src/util/def.hpp b/src/backend/graph_compiler/core/src/util/def.hpp
index d75299b..f754018 100644
--- a/src/backend/graph_compiler/core/src/util/def.hpp
+++ b/src/backend/graph_compiler/core/src/util/def.hpp
@@ -32,7 +32,7 @@
 #define SC_API __attribute__((visibility("default")))
 #endif
 
-#define SC_INTERNAL_API
+#define SC_INTERNAL_API SC_API
 
 #ifndef SC_MEMORY_LEAK_CHECK
 #define SC_MEMORY_LEAK_CHECK 0
